package.json

```
@@ -965,6 +965,12 @@
                     "description": "Set the root directory for loading files for the Python Interactive window.",
                     "scope": "resource"
                 },
+                "python.dataScience.forceJupyterExactMatch": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Force the Python Interactive window to use the python selected and don't search for a close match",
```

Why do we need to add `and don't search for a close match`? If we force the Python Interactive window to use a specific interpreter, no further information is required here I think?

Also, Python is capitalized.

```suggestion
                    "description": "Force the Python Interactive window to use the Python interpreter selected.",
```

---

Is python not supposed to be capitalized? It's the name of a window.

---

'and don't search for a close match' is necessary because without this flag it does use the actual python selected first but if that fails it falls back to searching through all other available pythons. I think if it just says Force it to use the selected python, user's will think it doesn't use the selected python at all without this check.

Maybe I invert it and make it 

'Search all installed pythons for a Jupyter installation when starting the Python Interactive window' and the default would then be true.